lib/filesystem.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Filesystem do
```

Comment:
```suggestion
defmodule FSS do
```

There is already a project called file_system and I think calling it filesystem would be confusing :)